src/tensor/TensorConversion.c

```
@@ -374,6 +330,74 @@ void convertAsymTensorToSymInt32Tensor(tensor_t *inputTensor, tensor_t *outputTe
+
+void convertSymTensorToAsymTensor(tensor_t *inputTensor, tensor_t *outputTensor) {
+    size_t n = calcNumberOfElementsByTensor(inputTensor);
+    symQConfig_t *inQC = inputTensor->quantization->qConfig;
+    int32_t mant[n];
+    unpackSignExtend(inputTensor->data, inQC->qBits, mant, n);
+    float deq[n];
+    for (size_t i = 0; i < n; i++) {
+        deq[i] = (float)mant[i] * inQC->scale;
+    }
+    asymQConfig_t *outQC = outputTensor->quantization->qConfig;
+    quantizeFloatToAsym(deq, n, outQC, outputTensor->data);
```

Cppcheck (2.21.0)

[warning] Uninitialized variable

(uninitvar)